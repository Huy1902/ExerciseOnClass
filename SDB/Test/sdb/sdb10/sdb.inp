100
84
38
1
35
59
23
91
27
56
21
79
65
72
42
20
36
15
85
7
87
10
17
65
87
89
32
51
61
19
8
3
90
33
86
1
94
27
70
5
17
52
44
70
54
49
81
36
55
59
85
11
38
19
4
78
78
76
50
37
22
23
47
67
96
46
76
42
7
37
20
64
65
92
94
7
36
78
3
53
77
83
72
50
64
32
13
20
70
38
5
1
28
14
87
30
91
84
37
83
67
