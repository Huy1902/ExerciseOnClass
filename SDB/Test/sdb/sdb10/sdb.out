32
35
56
21
79
15
10
89
51
61
8
90
33
86
52
44
54
49
81
55
11
4
22
47
96
46
92
53
77
13
28
14
30
